config/default.py

```diff
@@ -39,6 +39,7 @@ class Default:
     GENMEDIA_BUCKET = os.environ.get("GENMEDIA_BUCKET")
     IMAGE_FIREBASE_DB = os.environ.get("IMAGE_FIREBASE_DB", "")
     IMAGE_COLLECTION_NAME = os.environ.get("IMAGE_COLLECTION_NAME", "arena_images")
+    STUDY_COLLECTION_NAME = os.environ.get("STUDY_COLLECTION_NAME", "arena_study")
     IMAGE_RATINGS_COLLECTION_NAME = os.environ.get("IMAGE_RATINGS_COLLECTION_NAME", "arena_elo")
     ELO_K_FACTOR = int(os.environ.get("ELO_K_FACTOR", 32))
 
```

Comment:
We can add the prompt file here as a Config and then update the env_template with it as well

Reply:
Yes, my bad. The `gs://n25-vertex-ai-demos-1-genmedia/prompts/_alt_final_prompts.json` is no longer hard coded, it is based on the study the user selects(dynamically fetched from the firestore collection). 

When it comes to default/live generation, yes I can add it as a env variable